app/src/main/java/com/github/sdp/tarjetakuna/model/User.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.sdp.tarjetakuna.model
+
+import com.github.sdp.tarjetakuna.database.DBMagicCard
+
+/**
+ * Represents a user.
+ */
+data class User(
+    val email: String,
+    val username: String,
+    val cards: List<DBMagicCard>,
+    val location: Coordinates
+) {
+    init {
+        require(email.isNotBlank()) { "Username cannot be blank" }
```

Comment:
Is this the right comment to display? Maybe "user needs at least an email" is better as this is not the username field